router-bridge/src/plan.rs

```diff
@@ -225,7 +361,7 @@ mod tests {
     }
 
     #[test]
-    fn invalid_query_is_caught() {
+    fn syntactically_incorrect_query_is_caught() {
```

Comment:
I renamed this just to add clarity — since this is a parsing error, rather than a validation error.